app/views/providers/check_client_details/show.html.erb

```diff
@@ -1,41 +1,65 @@
+
+    <h2 class="govuk-heading-m"><%= t(".h2_client") %></h2>
+
+    <%= govuk_summary_list do |summary_list| %>
+      <% summary_list.with_row do |row| %>
+        <% row.with_key { t(".name") } %>
+        <% row.with_value { "#{@applicant.first_name} #{@applicant.last_name}" } %>
+        <% row.with_action(text: "Change", visually_hidden_text: t(".name"),
+                           href: providers_legal_aid_application_applicant_details_path(@legal_aid_application, checking_details_for_dwp)) %>
+       <% end %>
+       <% summary_list.with_row do |row| %>
+        <% row.with_key(text: t(".date_of_birth")) %>
+        <% row.with_value(text: @applicant.date_of_birth.strftime("%e %B %Y")) %>
+        <% row.with_action(text: "Change", visually_hidden_text: t(".date_of_birth"),
+                           href: providers_legal_aid_application_applicant_details_path(@legal_aid_application, checking_details_for_dwp)) %>
+      <% end %>
+       <% summary_list.with_row do |row| %>
+        <% row.with_key(text: t(".national_insurance_number")) %>
+        <% row.with_value(text: @applicant.national_insurance_number.gsub(/(.{2})(?=.)/, "\1 \2")) %>
+        <% row.with_action(text: "Change", visually_hidden_text: t(".national_insurance_number"),
+                           href: providers_legal_aid_application_has_national_insurance_number_path(@legal_aid_application, checking_details_for_dwp)) %>
+      <% end %>
+    <% end %>
+
+    <% if Setting.partner_means_assessment? && @legal_aid_application.partner.shared_benefit_with_applicant? %>
+      <h2 class="govuk-heading-m"><%= t(".h2_partner") %></h2>
+
+      <%= govuk_summary_list do |summary_list| %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key { t(".name") } %>
+          <% row.with_value { "#{@partner.first_name} #{@partner.last_name}" } %>
+          <% row.with_action(text: "Change", visually_hidden_text: t(".name"),
+                             href: providers_legal_aid_application_partners_details_path(@legal_aid_application)) %>
+        <% end %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".date_of_birth")) %>
+          <% row.with_value(text: @partner.date_of_birth.strftime("%e %B %Y")) %>
+          <% row.with_action(text: "Change", visually_hidden_text: t(".date_of_birth"),
+                             href: providers_legal_aid_application_partners_details_path(@legal_aid_application)) %>
+        <% end %>
+         <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".national_insurance_number")) %>
+          <% row.with_value(text: @partner.national_insurance_number.gsub(/(.{2})(?=.)/, "\1 \2")) %>
+          <% row.with_action(text: "Change", visually_hidden_text: t(".national_insurance_number"),
+                             href: providers_legal_aid_application_partners_details_path(@legal_aid_application)) %>
+        <% end %>
+       <% end %>
+    <% end %>
+
+    <br>
```

Comment:
nit: for layout purposes, it may be best to use [govuk spacing](https://design-system.service.gov.uk/styles/spacing/#static-spacing) overrides on either  sibling elements